.github/workflows/copilot-chat-tests.yml

```diff
@@ -0,0 +1,75 @@
+name: Copilot Chat Tests
```

Comment:
It's because we don't have the GitHub secrets yet. I am still waiting on Shawn for that.

Reply:
I removed the PR trigger since GitHub doesn't allow secrets to be passed to actions triggered by PRs. The tests can only be run after it's merged since they need the secrets.